docs/src/index.md

```diff
@@ -496,26 +474,33 @@ x_thread = @chain df begin
 end
 ```
 
-Another alternative is Pipe.jl which exports the `@pipe` macro for piping. 
-The piping mechanism in Pipe requires explicit specification of the piped
-object via `_` instead of assuming it is the first argument to the next function.
-The Pipe.jl equivalent of the above is:
+By default, `@chain` places the value of the 
+previous expression into the first argument of the current
+expression. The placeholder `_` is used to break that convention
+and refer to the argument returned from the previous 
+expression.
 
 ```julia
-using Pipe, Statistics
+@chain df begin 
+    map(_.a, _.x) do a, x
+        a + x
+    end
+end
+```
 
-df = DataFrame(a = repeat(1:5, outer = 20),
-               b = repeat(["a", "b", "c", "d"], inner = 25),
-               x = repeat(1:20, inner = 5))
+`@chain` also provides the `@aside` macro-flag to perform opertaions
+in the middle of a `@chain` block. 
 
-x_thread = @pipe df |>
-    @transform(_, y = 10 * :x) |>
-    @where(_, :a .> 2) |>
-    @by(_, :b, meanX = mean(:x), meanY = mean(:y)) |>
-    @orderby(_, :meanX) |>
-    @select(_, :meanX, :meanY, var = :b)
+```julia
+@chain df begin 
+    @transform y = 10 .* :x
+    @aside y_mean = mean(_.y)
```

Comment:
Mention `@aside` somewhere? Otherwise one might think it's a DataFramesMeta macro.

Reply:
Added a clarifying comment.